app/views/groups/show.html.slim

```diff
@@ -12,7 +12,7 @@
     span.glyphicon.glyphicon-menu-left
   = link_to group_path(@group, display_date: @display_date + 1.day), class: 'date-navigation date-navigation-right' do
     span.glyphicon.glyphicon-menu-right
-  .task-date-display= @display_date.to_s :long_ordinal
+  .task-date-display= @display_date.strftime("%a %b #{time.day.ordinalize}")
```

Comment:
You still need to swap out another variable from the stack overflow example:

  .task-date-display= @display_date.strftime("%a %b #{@display_date.day.ordinalize}")